### PR TITLE
New version of feng-shui

### PR DESCRIPTION
**Pull request by nczita**
Fix payouts checks

### CODE REVIEW

File: config/rubocop_default_rails_5.2.yml [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+  # by overriding DisplayCopNames, or by giving the -D/--display-cop-names
+  # option.
+  DisplayCopNames: true
+  # Style guide URLs are not displayed in offense messages by default. Change
+  # behavior by overriding DisplayStyleGuide, or by giving the
+  # -S/--display-style-guide option.
+  DisplayStyleGuide: true
+  # What version of the Ruby interpreter is the inspected code intended to
+  # run on? (If there is more than one, set this to the lowest version.)
+  TargetRubyVersion: 2.3
```
Comment (przprz) [inline config/rubocop_default_rails_5.2.yml]: Do we still have it anywhere?

File: lib/ah/feng_shui/version.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 module Ah
   module FengShui
-    VERSION = '0.1.9'.freeze
+    VERSION = '0.1.10'.freeze
```
Comment (przprz) [inline lib/ah/feng_shui/version.rb]: I suppose it deserves a bump of minor, doesn't it?

Comment (nczita) [reply lib/ah/feng_shui/version.rb]: I don't know - this is used anywhere? We just using GIT SHA anyway ;)

Comment (przprz) [reply lib/ah/feng_shui/version.rb]: Yeah, but since you bump it it'd be better to bump properly :P

**Pull request merged**